src/KeyboardApp.js

```diff
@@ -0,0 +1,67 @@
+import Keyboard from './Keyboard.js';
+import Key from './Key.js';
+
+export default class KeyboardApp {
+  constructor() {
+    this.board = new Keyboard();
+  }
+
+  init(elemToAppend) {
+    if (window.localStorage.getItem('lang')) {
+      this.board.isEngLang = window.localStorage.getItem('lang') === 'true';
```

Comment:
better is to use string parameter instead of boolean. Think about extensibility. If there will be a new change request to develop 3 languages - you will have to use quantum computer with 3 states